README.md

```diff
@@ -1,16 +1,13 @@
 PERMON toolbox
 ================================
-PERMON (Parallel Efficient Robust Modular Object Numerical) is a software toolbox for quadratic programming (QP) based on [PETSc](http://www.mcs.anl.gov/petsc/). It also includes domain decomposition methods (FETI and Total FETI) as a specific QP implementation.
+PERMON (Parallel Efficient Robust Modular Object Numerical) is a software toolbox for quadratic programming (QP) based on [PETSc](http://www.mcs.anl.gov/petsc/). It also includes domain decomposition methods (FETI and Total FETI) and Support Vector Machines (SVM) machine learning methods.
```

Comment:
It should be clear that this repository only contains QP stuff and FETI

Reply:
Would this suffice? https://github.com/permon/permon/pull/14/files#r417501398

Reply:
It is probably sufficient that it is in the repository description.

I suggest changing the repository description to  "PERMON main package for quadratic programming (PermonQP)".